com.github.sakya.corechess.json

```diff
@@ -0,0 +1,111 @@
+{
+  "app-id": "com.github.sakya.corechess",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "21.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.dotnet6"
+  ],
+  "command": "CoreChess",
+  "finish-args": [
+    "--filesystem=home",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--device=dri"
+  ],
+  "modules": [
+    {
+      "name": "CoreChess",
+      "buildsystem": "simple",
+      "build-options": {
+        "append-path": "/usr/lib/sdk/dotnet6/bin",
+        "append-ld-library-path": "/usr/lib/sdk/dotnet6/lib",
+        "env": {
+          "PKG_CONFIG_PATH": "/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet6/lib/pkgconfig"
+        },
+        "arch": {
+          "x86_64": {
+            "env": {
+              "RUNTIME": "linux-x64"
+            }
+          }
+        }
+      },
+      "build-commands": [
+        "dotnet publish CoreChess.sln -c Release --source ./nuget-sources --runtime $RUNTIME -p:PublishReadyToRun=true --self-contained --output ./dist/$RUNTIME",
+        "cp CoreChess/Lib/Bass/$RUNTIME/libbass.so ./dist/$RUNTIME/",
+        "cp -r --remove-destination ./dist/$RUNTIME /app/bin/",
+        "mkdir -p /app/share/applications/",
+        "cp ./Flatpak/com.github.sakya.corechess.desktop /app/share/applications/",
+        "mkdir -p /app/share/metainfo/",
+        "cp ./Flatpak/com.github.sakya.corechess.appdata.xml /app/share/metainfo/",
+        "mkdir -p /app/share/icons/hicolor/512x512/apps/",
+        "cp ./icon.png /app/share/icons/hicolor/512x512/apps/com.github.sakya.corechess.png"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "tag": "0.9.4.1",
+          "url": "https://github.com/sakya/corechess.git"
+        },
+        "nuget.json"
+      ]
+    },
+    {
+      "name": "Stockfish",
```

Comment:
Is there a reason why it's not being built from source? It doesn't seem complicated: Is there reason why it's not being build